tests/integration/testdata/sync/038-avoid-entity-rewrite-with-id-on-select-tag-mismatch/ca-certificate-initial.yaml

```diff
@@ -0,0 +1,33 @@
+_format_version: "3.0"
+ca_certificates:
+- cert: |-
+    -----BEGIN CERTIFICATE-----
```

Comment:
I've generated this locally, and has validity up to ~2030 - beyond which the test will fail and this needs update.

Reply:
Can you add a comment in the file to state the same? It will be easier to track that way.